### DISCUSSION TITLE
go-cqhttp发送群消息失败【已解决】

### DISCUSSION THREAD

**Discussion by huang122124** [Mrs4s/go-cqhttp#1447] [Q&A] [answered]
**### 发送群消息**
终结点：/send_group_msg
有一点很不解，发送私聊正常，发布群公告也正常，为什么发送群消息，一直是SEND_MSG_API_ERRO。该号群主、管理员身份均已尝试过，同样返回该错误。是不是哪个地方出问题了？
<img width="852" alt="螢幕截圖 2022-03-30 19 07 19" src="https://user-images.githubusercontent.com/44934948/160821042-8d4e7673-1e10-4e85-89e4-30916dc7fa10.png">

**Comment by qianjunakasumi** [comment]
请提交 gocqhttp 控制台日志

**Comment by huang122124** [comment, marked as answer]
[2022-03-30 19:19:23] [WARNING]: 群消息发送失败: 账号可能被风控.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Mrs4s/go-cqhttp"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月30日(星期三) 晚上7:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Mrs4s/go-cqhttp] go-cqhttp发送群消息失败 (Discussion #1447)





 
请提交 gocqhttp 控制台日志

**Reply by huang122124** [reply]
知道了，要等3-7天，谢谢了
